src/components/Ruler/index.js

```diff
@@ -44,8 +40,9 @@ const calculateY = (points, y) => {
 };
 
 const Ruler = ({ ruler, points, height, width }) => {
-  const yLabelHeight = height - labelHeight - 5;
-  const pointsObject = calculateY(points, yLabelHeight);
+  const yTooltipMargin = 5;
+  const yTooltipPosition = height - labelHeight - yTooltipMargin;
```

Comment:
I'm still not understanding why this is called `yTooltipPosition`? It seems like `pointsObject` is actually the tooltip positions, right? Is this just the available region that is able to be used when positioning tooltips?

Reply:
It is just a position of timestamp tooltip which suppose to stick to the most bottom position. Please check the Ruler story.